<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>FormFieldEditor</class>
    <widget class="QDialog" name="FormFieldEditor">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>955</width>
                <height>775</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>500</width>
                <height>400</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Field editor</string>
        </property>
        <layout class="QVBoxLayout" name="Layout_Vertical_1">
            <property name="spacing">
                <number>10</number>
            </property>
            <property name="leftMargin">
                <number>10</number>
            </property>
            <property name="topMargin">
                <number>10</number>
            </property>
            <property name="rightMargin">
                <number>10</number>
            </property>
            <property name="bottomMargin">
                <number>10</number>
            </property>
            <item>
                <widget class="QTableWidget" name="TableWidget_Field">
                    <property name="verticalScrollBarPolicy">
                        <enum>Qt::ScrollBarAlwaysOn</enum>
                    </property>
                    <property name="horizontalScrollBarPolicy">
                        <enum>Qt::ScrollBarAlwaysOn</enum>
                    </property>
                    <property name="editTriggers">
                        <set>QAbstractItemView::NoEditTriggers</set>
                    </property>
                    <property name="tabKeyNavigation">
                        <bool>false</bool>
                    </property>
                    <property name="selectionMode">
                        <enum>QAbstractItemView::SingleSelection</enum>
                    </property>
                    <property name="verticalScrollMode">
                        <enum>QAbstractItemView::ScrollPerPixel</enum>
                    </property>
                    <property name="horizontalScrollMode">
                        <enum>QAbstractItemView::ScrollPerPixel</enum>
                    </property>
                    <attribute name="horizontalHeaderVisible">
                        <bool>false</bool>
                    </attribute>
                    <attribute name="horizontalHeaderDefaultSectionSize">
                        <number>15</number>
                    </attribute>
                    <attribute name="horizontalHeaderMinimumSectionSize">
                        <number>15</number>
                    </attribute>
                    <attribute name="verticalHeaderVisible">
                        <bool>false</bool>
                    </attribute>
                    <attribute name="verticalHeaderDefaultSectionSize">
                        <number>15</number>
                    </attribute>
                    <attribute name="verticalHeaderMinimumSectionSize">
                        <number>15</number>
                    </attribute>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="HBoxLayout_1">
                    <property name="spacing">
                        <number>10</number>
                    </property>
                    <item>
                        <widget class="QFrame" name="Frame_Tools">
                            <property name="minimumSize">
                                <size>
                                    <width>200</width>
                                    <height>30</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>200</width>
                                    <height>30</height>
                                </size>
                            </property>
                            <property name="frameShape">
                                <enum>QFrame::Panel</enum>
                            </property>
                            <property name="frameShadow">
                                <enum>QFrame::Plain</enum>
                            </property>
                            <widget class="QRadioButton" name="RadioButton_ToolNone">
                                <property name="geometry">
                                    <rect>
                                        <x>10</x>
                                        <y>3</y>
                                        <width>65</width>
                                        <height>24</height>
                                    </rect>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>65</width>
                                        <height>24</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>65</width>
                                        <height>24</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>[None]</string>
                                </property>
                                <property name="checked">
                                    <bool>true</bool>
                                </property>
                            </widget>
                            <widget class="QRadioButton" name="RadioButton_ToolOn">
                                <property name="geometry">
                                    <rect>
                                        <x>80</x>
                                        <y>3</y>
                                        <width>55</width>
                                        <height>24</height>
                                    </rect>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>55</width>
                                        <height>24</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>55</width>
                                        <height>24</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>[On]</string>
                                </property>
                            </widget>
                            <widget class="QRadioButton" name="RadioButton_ToolOff">
                                <property name="geometry">
                                    <rect>
                                        <x>140</x>
                                        <y>3</y>
                                        <width>55</width>
                                        <height>24</height>
                                    </rect>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>55</width>
                                        <height>24</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>55</width>
                                        <height>24</height>
                                    </size>
                                </property>
                                <property name="text">
                                    <string>[Off]</string>
                                </property>
                            </widget>
                        </widget>
                    </item>
                    <item>
                        <spacer name="Spacer_Horizontal_1">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="PushButton_Ok">
                            <property name="minimumSize">
                                <size>
                                    <width>70</width>
                                    <height>30</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>70</width>
                                    <height>30</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>OK</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="PushButton_Cancel">
                            <property name="minimumSize">
                                <size>
                                    <width>70</width>
                                    <height>30</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>70</width>
                                    <height>30</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>Cancel</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <tabstops>
        <tabstop>TableWidget_Field</tabstop>
        <tabstop>RadioButton_ToolNone</tabstop>
        <tabstop>RadioButton_ToolOn</tabstop>
        <tabstop>RadioButton_ToolOff</tabstop>
        <tabstop>PushButton_Ok</tabstop>
        <tabstop>PushButton_Cancel</tabstop>
    </tabstops>
    <resources/>
    <connections/>
</ui>
